### ISSUE TITLE
Category appears under tags

### ISSUE THREAD

**Issue by talha131** [Pelican-Elegant/elegant#283]
It shouldn't be there. We have a specific row for it. Clicking on it takes to all tags page which is wrong.

![](https://i.imgur.com/FAvhO2W.png)

**Comment by iranzo** [comment]
![image](https://user-images.githubusercontent.com/312463/59157525-4483b100-8aac-11e9-8eb3-43dc4c17d01e.png)


Works fine for me with current 'master' templates:


~~~yaml
---
author: Pablo Iranzo Gómez (pablo.iranzo@gmail.com)
title: Use custom domain name with Blog-O-Matic
tags: pelican, foss, travis, ci/cd, elegant, blog, python, github, blog-o-matic, linux
layout: post
date: 2019-05-16 22:29:14 +0200
comments: true
category: tech
description:
---
~~~

**Comment by talha131** [comment]
No, the bug is still present and my guess is that this bug is in Pelican code. 

I tried default theme

![](https://i.imgur.com/G8HUowA.png)

Notice that "Monetization" which is a category is present in tags field.

We may have to debug pelican code.

**Comment by iranzo** [comment]
I guess it's on pelican, the one we render at https://pelican-elegant.github.io/amazon-onelink (I guess it's the one you use as sample) is not showing it when using latest pelican generator

**Comment by talha131** [comment]
That's strange. I am using 

```
pelican --version
4.0.1
```

I am going to investigate this in a fresh env.

**Comment by iranzo** [comment]
What I'm using:

~~~
Package         Version
--------------- -------
beautifulsoup4  4.7.1  
blinker         1.4    
bs4             0.0.1  
cssmin          0.2.0  
cssutils        1.0.2  
docopt          0.6.2  
docutils        0.14   
feedgenerator   1.9    
ghp-import      0.5.5  
Jinja2          2.10   
Markdown        3.0.1  
MarkupSafe      1.1.0  
pandoc          1.0.2  
pelican         4.0.1  
peru            1.2.0  
piexif          1.1.2  
Pillow          6.0.0  
pip             19.1.1 
ply             3.11   
Pygments        2.3.1  
pypandoc        1.4    
pysvg           0.2.2  
python-dateutil 2.7.5  
pytz            2018.9 
PyYAML          3.13   
setuptools      41.0.1 
six             1.12.0 
smartypants     2.0.1  
soupsieve       1.7.1  
typogrify       2.0.7  
Unidecode       1.0.23 
webassets       0.12.1 
wheel           0.32.3 
~~~

That's from my laptop, virtualenv on website is regenerated each time with 'latest' versions available

**Comment by talha131** [comment]
Thank you. I will compare.

**Comment by talha131** [comment]
Fixed via aae403ef046b77c204bf2669aff475872b10dfba

Apparently it is not enough to update Pelican. You also need to have the updated Makefile
